### DISCUSSION TITLE
Clearing previously set color filters

### DISCUSSION THREAD

**Discussion by asherLZR** [airbnb/lottie-ios#1741] [General]
Good day, is there a way to clear color filters in lottie-ios? In lottie-android, this looks something like the following, captured by the test case [here](
https://github.com/airbnb/lottie-android/blob/master/snapshot-tests/src/androidTest/java/com/airbnb/lottie/snapshots/tests/DynamicPropertiesTestCase.kt#L295).

```
view.addValueCallback(new KeyPath("**"), LottieProperty.COLOR_FILTER, new LottieValueCallback<>(null));
```

**Comment by calda** [comment]
It seems like this isn't currently possible, but could be a reasonable to add. What's your use case for removing a previously-set color provider?

**Reply by asherLZR** [reply]
Thanks for checking. We use lottie-react-native with components which apply the `colorFilter` property dynamically based on component state (e.g. a theme change, among others). If the state changes from `colorFilter=[{ path: "*",  color: "black" }]` to `colorFilter=[]/undefined` we would expect that the colorFilter is cleared, but the actual behaviour is that it persists in the old state. Having this control on lottie-ios would allow us to address this.

**Comment by Alex-Dobrynin** [comment]
any update on this issue?
